Breeze.UI/src/app/shared/components/generic-modal/generic-modal.component.ts

```diff
@@ -9,8 +9,9 @@ import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
 })
 export class GenericModalComponent implements OnInit {
 
-  @Input() public title: string = "Something went wrong";
-  @Input() public body: string = "Something went wrong while connecting to the API. Please restart the application.";
+  @Input() public title = 'Something went wrong';
```

Comment:
Got rid of the string types. Why? You do this as well later on with a boolean

Reply:
https://palantir.github.io/tslint/rules/no-inferrable-types/